docs/hero-banner.md

```diff
@@ -14,7 +14,7 @@ Here are some cool activities to get you started!
 ---
 * name: Build a Bubble Popping Game
 * buttonLabel: Code Now
-* description: Create your own bubble popping game with the crew from Disney & Pixar's Inside Out 2!
+* description: Code with images from Disney & Pixar's Inside Out 2!
```

Comment:
I'm okay with this, but I'll note that it does sound a little like we're using the images to write actual code (as opposed to blocks)

Reply:
I agree.. I like the little crew sentiment in there, I think that's cute. Could it be changed to something like `Code with the crew (maybe emotions here?) from Disney and Pixar's Inside Out 2`? It's a bit of a nod to the first pop-up, too, since the pop-up says "are you ready to try something new with Joy and the rest of the emotions..."

Reply:
Yeah, I tried to figure out how to leave it in, but I think it's still too long.  Since the banner already has the Disney/Pixar stuff in it, we could probably just say "Code a game featuring some of our favorite characters." Because, technically, I need to get everything approved by Disney/Pixar if we use their brand.

Reply:
Maybe "Code a game featuring characters from the new movie!"

Reply:
I like that a lot!!